Mathlib/Combinatorics/SimpleGraph/Basic.lean

```diff
@@ -550,6 +549,10 @@ instance fintypeEdgeSetSdiff [DecidableEq V] [Fintype G₁.edgeSet] [Fintype G
   rw [edgeSet_sdiff]
   exact Set.fintypeDiff _ _
 
+theorem card_top_edgeSet [DecidableEq V] [Fintype V] :
```

Comment:
I believe this is [SimpleGraph.card_edgeFinset_top_eq_card_choose_two](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Combinatorics/SimpleGraph/Finite.html#SimpleGraph.card_edgeFinset_top_eq_card_choose_two)

Reply:
Ah wait, there is difference with edgeFinset versus edgeSet. You could do `rw [← edgeFinset_card, SimpleGraph.card_edgeFinset_top_eq_card_choose_two])` in `card_EdgeLabelling` though instead.